integration_test/Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN apt-get update && apt-get install -y \
   python3 python3-pip \
   git yosys
 
-RUN python3 -m pip install pycapnp psutil
+RUN python3 -m pip install pycapnp psutil pybind11~=2.6 numpy
```

Comment:
pybind11 we obviously need, and at that version. adding numpy because MLIR wants it: https://github.com/llvm/llvm-project/blob/ea659ea101a56a08c73be65ab62c3e180e37ca38/mlir/CMakeLists.txt#L109. Not exactly sure why.